src/providers/aws.rs

```diff
@@ -0,0 +1,858 @@
+use crate::ssh;
+use crate::Machine;
+use failure::{Error, ResultExt};
+use itertools::Itertools;
+use rusoto_core::request::HttpClient;
+use rusoto_core::{ProvideAwsCredentials, Region};
+use rusoto_ec2::Ec2;
+use std::collections::HashMap;
+use std::io::Write;
+use std::sync::Arc;
+use std::{thread, time};
+
+struct UbuntuAmi(String);
```

Comment:
This type, or at least its implications for defaults, should be documented somewhere.

Reply:
It is: https://github.com/jonhoo/tsunami/pull/22/files#diff-60699702750aef5a8cdc050b4fce64eaR120

Reply:
Users shouldn't otherwise have to know about this type I think.